FirebaseStorageSwift/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Unreleased
+
+# v0.1
+- Initial public beta release. Extend the Storage Reference API with the Swift
+  Result Enum type for all APIs that return an optional type and optional Error.
```

Comment:
nit: `Extend` -> `Extends`, `Result Enum` -> `Result`, and `optional type` -> `optional value`